package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "fh-wfm-user-angular",
+  "version": "0.1.0",
+  "description": "An AngularJS module for viewing User details.",
+  "main": "lib/user-ng.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/feedhenry-raincatcher/raincatcher-user.git"
+  },
+  "scripts": {
+    "build": "wfm-template-build -m 'wfm.user.directives' -t lib/template",
```

Comment:
No need to add old aproach to new modules? Let's use grunt wfm-template:watch etc :)